devU-client/src/components/authenticatedRouter.tsx

```diff
@@ -35,18 +35,18 @@ const AuthenticatedRouter = () => (
     <Route exact path='/users/:userId/assignments' component={HomePage} />
```

Comment:
I would suggest for main pages do not contain usersId since it is not coherent, it will be from user/userId suddenly go to /course/courseId and since userid is stored in const userId = useAppSelector((store) => store.user.id)